src/Controller/MenuController.java

```diff
@@ -258,7 +266,6 @@ public void addActivity() {
 			if (activity != null) {
 				MainController.getSPC().addActivity(activity);
 			}
-			openMenu.fire();
```

Comment:
Can you explain why this line was removed?

Reply:
This line was removed because, originally this item toggled the hamburger menu when clicking the "Add Activity" button, which was located inside the hamburger menu. Since it was moved outside, the button when clicked caused the hamburger menu to open if it wasn't open. So I removed it, because the hamburger menu opened for no reason.

Reply:
OK.  Thanks for the explanation.